src/main/drivers/pwm_output.c

```diff
@@ -242,6 +251,36 @@ void pwmFastPwmMotorConfig(const timerHardware_t *timerHardware, uint8_t motorIn
     motors[motorIndex]->pwmWritePtr = pwmWritePtr;
 }
 
+#ifdef USE_DIGITAL_MOTORS
+void pwmDigitalMotorConfig(const timerHardware_t *timerHardware, uint8_t motorIndex, motorPwmProtocolTypes_e pwmProtocolType)
+{
+	pwmDigitalMotorHardwareConfig(timerHardware, motorIndex, pwmProtocolType == PWM_TYPE_DSHOT600);
+    motors[motorIndex] = &pwmOutputPorts[allocatedOutputPortCount++];
+    motors[motorIndex]->pwmWritePtr = pwmWriteDigital;
+}
+#endif
+
+bool pwmMotorWriteConfig(motorPwmProtocolTypes_e motorProtocol, bool use_unsyncedpwm)
+{
+    pwmCompleteWritePtr = NULL;
+    switch (motorProtocol) {
+    case PWM_TYPE_ONESHOT125:
+    case PWM_TYPE_ONESHOT42:
+    case PWM_TYPE_MULTISHOT:
+        if (!use_unsyncedpwm) {
+            pwmCompleteWritePtr = pwmCompleteOneshotMotorUpdate;
+        }
+        return use_unsyncedpwm;
+#ifdef USE_DIGITAL_MOTORS        
```

Comment:
No motors are digital. All ESCs are digital. The INTERFACE can be analog or digital. 
I would say "USE_DSHOT".